Core/NoOpRunner.Core/Shapes/EntityShapes/Rocket.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace NoOpRunner.Core.Shapes.EntityShapes
 {
-    class Rocket : EntityShape
+    public class Rocket : EntityShape
```

Comment:
Don't make the classes public, use `InternalsVisibleTo` in AssemblyInfo.cs if need be instead

Reply:
These classes aren't supposed to be private. They should be public by default and only private explicitly if intended.